### PR TITLE
Searchfield style

### PR DESCRIPTION
**Pull request by Behnazz**
make the style of search field in the homepage and categories the same

### CODE REVIEW

File: frontend/src/Components/HomePage/HomePage.css [hunk 1]
```diff
@@ -39,7 +39,7 @@ form div div::before {
 .search-text,
 .postcode-field {
   width: 348px;
-  height: 40px;
+  /* height: 40px; */
```
Comment (Wolfjawan) [inline frontend/src/Components/HomePage/HomePage.css]: Please remove the code wich is not in use anymore

File: frontend/src/Components/HomePage/HomePage.css [hunk 2]
```diff
@@ -76,7 +76,7 @@ form div div::before {
   box-shadow: none !important;
 }
 
-.show-more,
+/* .show-more,
```
Comment (Wolfjawan) [inline frontend/src/Components/HomePage/HomePage.css]: same above

File: frontend/src/Components/Organisation/search.css [hunk 3]
```diff
@@ -40,23 +36,35 @@
...
+  border-radius: 3px;
+  border: 0.5px solid rgba(136, 137, 137, 0.68);
+  padding: 15px 20px 15px 3rem;
   width: 32vw;
+  color: rgba(50, 49, 49, 0.9);
+  font-weight: 600;
+  height: 11px;
+  background: #eeeeee;
+  margin-bottom: 20px !important;
 }
```
Comment (khaledkzy) [inline frontend/src/Components/Organisation/search.css]: A better way is to target more than one class if you can like this,
.org-search .post-code input, org-search .post-codesearch input {
}

File: frontend/src/Components/HomePage/HomePage.css (hunk 1 above)
Comment (dahfool) [reply frontend/src/Components/HomePage/HomePage.css]: @Behnazz please delete

**Comment by dahfool** [comment]
@Behnazz please fix conflicts

**Pull request merged**